user-manager-service/src/main/java/com/crmfoodestablishment/userauthservice/usermanager/entity/User.java

```diff
@@ -0,0 +1,55 @@
+package com.crmfoodestablishment.userauthservice.usermanager.entity;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.UUID;
+
+@Entity
+@Table(name = "users")
+@Getter
+@Setter
+@NoArgsConstructor
+public class User {
+
+    @Id
+    @SequenceGenerator(name="users_seq", sequenceName = "users_seq")
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "users_seq")
+    @Column(name = "id", nullable = false, unique = true)
+    private Long id;
+
+    @Column(name = "uuid", nullable = false, unique = true, updatable = false)
+    private UUID uuid;
+
+    @Column(name = "email", nullable = false, unique = true)
+    private String email;
+
+    @Column(name = "password", nullable = false)
+    private String password;
+
+    @Column(name = "role", nullable = false)
+    @Enumerated(EnumType.STRING)
+    private Role role;
```

Comment:
If u are going to have only 1 role you need to have ideas how to implement extended 'roles' or 'business functions'
Let's discuss it

Reply:
Resolved: we have decided to have 1 role at the current state in the project

Reply:
resolved